### DISCUSSION TITLE
Multi Tenancy Configuration Issue

### DISCUSSION THREAD

**Discussion by atish23** [saml-idp/saml_idp#222] [General]
I have a multi-tenant application that can support multiple SP configurations for each tenant.
So, for example, we will need a different single_service_post_location base url, and there is no way to pass it dynamically.
How can we achive this??
@Zogoo

**Comment by Zogoo** [comment]
@atish23 as for now this is not easy to do.
It would help if you overrode value each time a company-specific request comes to your server. 
The only thing you need to remember is that your server should run a web server with the process (not thread) until we shift config "instance variable".

**Comment by atish23** [comment]
Can i start working on this request? @Zogoo 
Will share approach and if it looks good then we can work on it.
